snuba/admin/static/tracing/index.tsx

```diff
@@ -110,11 +117,25 @@ function TracingQueries(props: { api: Client }) {
     );
   }
 
-  function rawTraceDisplay(title: string, value: any): JSX.Element {
+  function rawTraceDisplay(title: string, value: any, profileEventResults: ProfileEvent): JSX.Element | undefined {
     const parsedLines: Array<string> = value.split(/\n/);
 
+    const profileEventRows: Array<string> = [];
+    for (const [k, v] of Object.entries(profileEventResults)) {
+      profileEventRows.push(v.rows[0]);
+    }
+
     return (
       <ol style={collapsibleStyle} key={title + "-root"}>
+        <h4>Profile Events Output</h4>
+        {profileEventRows.map((line, index) => {
+          return (
+            <li key={title + index}>
+              <span>{line}</span>
```

Comment:
```suggestion
              <Text>{line}</Text>
```

Mantine provides a Text component which is easier to style.